### ISSUE TITLE
Checkstyle: don't fail if line attribute is not present

### ISSUE THREAD

**Issue by fouadchamoun** [tomasbjerre/violations-lib#107]
Hi !

I'm having trouble parsing scalastyle (checkstyle) files as some of the errors don't have the "line" attribute.
For example : 

```
java.lang.RuntimeException: "line" not found in "<error source="org.scalastyle.file.NewLineAtEofChecker" severity="error" message="File must end with newline character"/>"
        at shadow.se.bjurr.violations.lib.util.ViolationParserUtils.getAttribute(ViolationParserUtils.java:81)
        at shadow.se.bjurr.violations.lib.util.ViolationParserUtils.getIntegerAttribute(ViolationParserUtils.java:128)
        at shadow.se.bjurr.violations.lib.parsers.CheckStyleParser.parseReportOutput(CheckStyleParser.java:35)
        at shadow.se.bjurr.violations.lib.reports.Parser.findViolations(Parser.java:122)
        at shadow.se.bjurr.violations.lib.ViolationsApi.violations(ViolationsApi.java:85)
        at se.bjurr.violations.main.Runner.getAllParsedViolations(Runner.java:398)
        at se.bjurr.violations.main.Runner.performTask(Runner.java:270)
        at se.bjurr.violations.main.Runner.main(Runner.java:259)
        at se.bjurr.violations.main.Main.main(Main.java:6)
```

Could we make it optional like the "column" attribute ?

https://github.com/tomasbjerre/violations-lib/blob/28da15daf35478ca72256be269bee5804c5d2ed6/src/main/java/se/bjurr/violations/lib/parsers/CheckStyleParser.java#L35

Thank you 😃

**Comment by fouadchamoun** [comment]
btw, i'm using the library through `violations-command-line` npm package

**Comment by tomasbjerre** [comment]
Do you have a sample xml?

**Comment by fouadchamoun** [comment]
there you go:
'''
<?xml version="1.0" encoding="UTF-8"?>
<checkstyle version="5.0">
 <file name="/home/fchamoun/work/data-analytics/databricks/apps/data-streaming/src/main/scala/com/work/spark/dynamoDb/utils/KafkaDataReader.scala">
  <error source="org.scalastyle.file.NewLineAtEofChecker" severity="error" message="File must end with newline character"/>
 </file>
</checkstyle>
'''

**Comment by tomasbjerre** [comment]
released in 1.10 of violations-command-line

**Comment by fouadchamoun** [comment]
thank you very much @tomasbjerre !
